superset-frontend/plugins/plugin-chart-echarts/src/BoxPlot/types.ts

```diff
@@ -36,7 +36,9 @@ export type BoxPlotFormDataWhiskerOptions =
   | 'Tukey'
   | 'Min/max (no outliers)'
   | '2/98 percentiles'
-  | '9/91 percentiles';
+  | '5/95 percentiles'
+  | '9/91 percentiles'
+  | '10/90 percentiles';
```

Comment:
Silly question but these are just descriptive strings; what's actually applying the percentiles for these new options?

Reply:
I think those strings get pulled out here: https://github.com/apache/superset/blob/master/superset/utils/pandas_postprocessing/boxplot.py#L93

Reply:
The percentiles are regexed out of the string, so as long as long as the string is compliant it'll work: https://github.com/apache/superset/blob/acf91e1f609f2106d019aa40fcf66a7c147e36d0/superset-frontend/packages/superset-ui-chart-controls/src/operators/boxplotOperator.ts#L28

Reply:
Seems like ab it of a flaky way to handle it but it's not new code so fair enough :+1: